core/silkworm/types/block_test.cpp

```diff
@@ -188,4 +188,28 @@ TEST_CASE("EIP-1559 Header RLP") {
     CHECK(decoded == h);
 }
 
+TEST_CASE("Hash header boundary computation") {
+    BlockHeader h;
+    h.difficulty = intx::from_string<intx::uint256>("0x13009de5666753258eb9306157680dc5da0d");
```

Comment:
Intx has support for literals: `= 0x13009de5666753258eb9306157680dc5da0d_uint256`.

Reply:
how to enable intx literals?

Reply:
I believe with `using namespace intx;`

Reply:
`0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff_uint256` does not work

Reply:
it was _u256